cmd/sync/sync.go

```diff
@@ -12,40 +17,55 @@ import (
 	"github.com/jim-minter/azure-helm/pkg/plugin"
 )
 
-var dryRun = flag.Bool("dry-run", false, "Print resources to be synced instead of mutating cluster state")
+var (
+	dryRun = flag.Bool("dry-run", false, "Print resources to be synced instead of mutating cluster state.")
+	once   = flag.Bool("run-once", true, "If true, run only once then quit.")
+)
 
 func sync() error {
+	log.Print("Sync process started!")
+	log.Print("Reading _data/manifest.yaml...")
```

Comment:
Wrap is returning an error. Unclear how would I use it for printing logs.

Reply:
I'm suggesting that the level of verbosity here is excessive and it's best not to lead other engineers down the path of `print("doing x"); x(); print("doing y"); y()`...  Some logging is surely required, but it may be that by using errors.Wrap() we can avoid a lot of it by improving the quality of errors returned when they happen.  Could be a follow-up.